base/applications/kbswitch/kbswitch.c

```diff
@@ -55,12 +55,23 @@ vDbgPrintExWithPrefix(IN PCCH Prefix,
                       IN PCCH Format,
                       IN va_list ap)
 {
-    CHAR Buffer[512];
-    SIZE_T PrefixLength = strlen(Prefix);
-    strncpy(Buffer, Prefix, PrefixLength);
-    _vsnprintf(Buffer + PrefixLength, _countof(Buffer) - PrefixLength, Format, ap);
-    Buffer[_countof(Buffer) - 1] = ANSI_NULL; /* Avoid buffer overrun */
-    OutputDebugStringA(Buffer);
+    static CHAR s_szBuffer[256] = "";
+    SIZE_T cch;
+
+    UNREFERENCED_PARAMETER(Prefix);
```

Comment:
This Prefix thing could still be taken into account fine as wsa done in the old code.